### DISCUSSION TITLE
Completely disable rich formatting

### DISCUSSION THREAD

**Discussion by mgaitan** [Textualize/rich#2926] [Q&A]
I work with a colleague of the old guard, one of those experts in vim and who prefer lines of up to 80 characters. We get along well but we have different styles for our CLI scripts. I put from time to time an emoji in some message and from time to time he takes it out, in a comical and silent editing battle. When I decided to take the bull by the horns and raise the issue he gave me his argument. 

> I personally find them distracting in console output or documentation, and anything fun or jokey loses it's charm once I see it for the third time, so I like to keep things as boring as possible, (...) it's not just about emoji for me, I like documentation and CLI output to read more neutral and less like the developer is sharing their thoughts with me. The less output the better, so I can find what's important. I like the Unix philosophy of no news is good news, i.e. if nothing went wrong, there should ideally be no output at all.

So, as a gesture of fellowshipt I'd like to make the whole "fanfare" of rich configurable and allow him some ascetic peace of mind. 

Is there any way to completely disable rich formatting ? I know about `NO_COLOR` envvar and `Console`'s `no_color=True` and `emoji=False` options. but consider this example. 

![image](https://user-images.githubusercontent.com/2355719/232203350-95afef10-b363-4699-8e25-31739ebf9e6c.png)

Is there any configuration that simply gets a totally plain `I like pizza`  or at least ` I like pizza` (without stripping) ?

**Comment by willmcgugan** [comment]
Closest you will get is TERM=dumb

But it won’t strip emoji, which are treated no differently than other characters.
